homeassistant/components/met/config_flow.py

```diff
@@ -81,6 +87,14 @@ async def async_step_import(self, user_input: dict | None = None) -> dict[str, A
 
     async def async_step_onboarding(self, data=None):
         """Handle a flow initialized by onboarding."""
+        # Don't create entry if latitude or longitude isn't set.
+        # Also, filters out our onboarding default location.
+        if (not self.hass.config.latitude and not self.hass.config.longitude) or (
+            self.hass.config.latitude == DEFAULT_HOME_LATITUDE
+            and self.hass.config.longitude == DEFAULT_HOME_LONGITUDE
+        ):
+            return self.async_abort(reason="no_home")
```

Comment:
We should add the reason to the translation strings.

Reply:
Done!